<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpMenu</class>
 <widget class="QWidget" name="SignUpMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>661</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>263</y>
     <width>280</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border:2px solid dark-grey;
</string>
   </property>
   <property name="text">
    <string extracomment="Username"/>
   </property>
  </widget>
  <widget class="QPushButton" name="signupButton">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>590</y>
     <width>90</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:focus { outline-color:white; 
border:1px solid #ffffff;}
QPushButton{
background:transparent;
border:none;
color:white;
}
</string>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameLineEdit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>323</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border:2px solid dark-grey;

</string>
   </property>
   <property name="text">
    <string extracomment="Password"/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>544</x>
     <y>50</y>
     <width>192</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>EMAIL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>330</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;

</string>
   </property>
   <property name="text">
    <string>USERNAME</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastLineEdit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>203</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border:2px solid dark-grey;

</string>
   </property>
   <property name="text">
    <string extracomment="Password"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;

</string>
   </property>
   <property name="text">
    <string>LAST NAME</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstLineEdit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>143</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border:2px solid dark-grey;
</string>
   </property>
   <property name="text">
    <string extracomment="Username"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>FIRST NAME</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="confPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>443</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border:2px solid dark-grey;

</string>
   </property>
   <property name="text">
    <string extracomment="Password"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>450</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;

</string>
   </property>
   <property name="text">
    <string>CONFIRM PASSWORD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>383</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border:2px solid dark-grey;
</string>
   </property>
   <property name="text">
    <string extracomment="Username"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>390</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>PASSWORD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>490</y>
     <width>120</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="mRadioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">outline:none;
background-color:transparent;
color:white;</string>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="fRadioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">outline:none;
background-color:transparent;
color:#ffffff;</string>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>512</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>GENDER</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>512</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>AGE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="ageSpinBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>512</y>
     <width>48</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">outline:none;</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="emailValidateView">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>263</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:0px;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>570</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:focus { outline-color:white; 
border:1px solid #ffffff;}
QPushButton{
background:transparent;
border:none;
color:white;
}
</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>firstLineEdit</tabstop>
  <tabstop>lastLineEdit</tabstop>
  <tabstop>emailLineEdit</tabstop>
  <tabstop>usernameLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>confPasswordLineEdit</tabstop>
  <tabstop>ageSpinBox</tabstop>
  <tabstop>mRadioButton</tabstop>
  <tabstop>fRadioButton</tabstop>
  <tabstop>signupButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
